### DISCUSSION TITLE
Zen Glance by hovering the cursor over links

### DISCUSSION THREAD

**Discussion by realSilasYang** [zen-browser/desktop#2443] [Ideas]
The new feature Zen Glance allows users to preview links using methods like "ctrl+click" or "alt+click," which is indeed a great addition. However, it would be even more convenient if it could function similarly to Wikipedia, where users can preview links simply by hovering the cursor over them. This enhancement would significantly improve user experience.

Additionally, I've noticed that Zen Glance does not work on search results pages, such as Google Search. Given that these pages are often where users seek quick information, enabling Zen Glance in this context would be highly practical. It would be beneficial if this functionality could be considered for future updates.

**Comment by yasinkavakli** [comment]
Oh god please no. Hover functionality is for websites and not for the browser itself. So many websites who use the hover functionality and could potentially break.
Please do not add a functionality which is intended for websites to the browser itself.

**Reply by realSilasYang** [reply]
It's quite straightforward to enhance compatibility by detecting whether a website has hover functionality. If a site does not support this feature, Zen Glance could automatically initiate link previews. This approach would streamline the user experience and ensure that users can access previews without needing to rely on specific key combinations like "ctrl+click" or "alt+click." 

Implementing this feature would not only make Zen Glance more user-friendly but also align it with the expectations set by other platforms that utilize hover functionality effectively.

**Reply by tedbergstrand** [reply]
I'm with Yasin. Hovering over a link is often used to look for a discrepancy between the link text and the link URL. Force-loading a potentially bad URL isn't something I want. You could macro your modifier key to a mouse button if it's something you absolutely need to do from mouse-only. But it feels like a really bad idea to accidentally open links because the cursor lands there.
